pkg/jobs/config.go

```diff
@@ -18,6 +18,7 @@ import (
 const (
 	intervalBaseSettingKey     = "jobs.registry.interval.base"
 	adoptIntervalSettingKey    = "jobs.registry.interval.adopt"
+	maxAdoptionsPerLoopKey     = "jobs.registry.max_adoptions_per_loop"
```

Comment:
[nit] Seems odd to put this in the middle of the `.interval` settings. Mind moving it below? I suppose they both have to do with adoption, but we don't do the same for retention time or the cancel update limit.